### PR TITLE
Use UUID instead of (integer)Id

### PR DESCRIPTION
**Pull request by paniagua**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -763,8 +763,8 @@ The collection is paginated and sorted by id (descending).
             Accept: application/vnd.salemove.v1+json
 
 + Parameters
-    + operator_ids (required, array of numbers, `operator_ids[]=1&operator_ids[]=2&operator_ids[]=3`) ... The ids of operators whose engagements are requested.
-    + site_ids (required, array of numbers, `site_ids[]=1&site_ids[]=2&site_ids[]=3`) ... The ids of sites which engagements are requested.
+    + operator_ids (required, array of strings, `operator_ids[]=892d7f8a-a506-4222-9385-5d0bd44eb255&operator_ids[]=e1c6bd0b-aef2-4948-8182-85a075f419e0&operator_ids[]=e8ad3dc1-c2c7-45c5-970d-98aaa8d91a6e`) ... The ids of operators whose engagements are requested.
```
Comment (indrekj) [inline apiary.apib]: could we use `array of uuids` or do we want to mark them as strings? @deiwin

Comment (deiwin) [inline apiary.apib]: I think the fact that the strings are in UUID format is irrelevant externally. I say we use `string` in all documentation. And as far as I know, this is the norm elsewhere as well.

Comment (deiwin) [inline apiary.apib]: Elsewhere meaning in other popular APIs

**Comment by indrekj** [comment]
+1

**Pull request merged**